packages/mgt-chat/src/statefulClient/StatefulGraphChatClient.ts

```diff
@@ -1130,8 +1145,28 @@ detail: ${JSON.stringify(eventDetail)}`);
     // this._eventEmitter.on('chatThreadDeleted', this.onChatDeleted);
     this._eventEmitter.on('participantAdded', this.onParticipantAdded);
     this._eventEmitter.on('participantRemoved', this.onParticipantRemoved);
+    this._eventEmitter.on('graphNotificationClientError', this.onGraphNotificationClientError);
   }
 
+  /**
+   * Handles the GraphNotificationClient emitted errors.
+   */
+  private readonly onGraphNotificationClientError = (e: Error) => {
+    this.updateErrorState(e);
+  };
+
+  /**
+   * Emits the error state to the stateful client.
+   */
+  private readonly updateErrorState = (e: Error) => {
+    error(e?.message);
```

Comment:
This should have a default error message to go to the console on the off chance that e?.message is falsy
```suggestion
    if (e?.message) {
      error(e?.message);
    } else {
      error('an unknown error occurred', e);
    }
```